### PR TITLE
[zh] add the missing content in [zh/docs/setup/production-environment/container-runtimes.md]

### PR DESCRIPTION
**Pull request by Yuaninga**
Supplement the Chinese version of containerd installed by windows when running the container of community production environment

### CODE REVIEW

**Comment by Yuaninga** [comment]
Application for personal certification

**Comment by Yuaninga** [comment]
Application for personal certification again

File: content/zh/docs/setup/production-environment/container-runtimes.md [hunk 1]
```diff
@@ -843,6 +843,39 @@ systemctl restart containerd
...
+# cni的的二进制文件和配置文件路径
+# bin_dir = "C:\\Program Files\\containerd\\cni\\bin"
+# conf_dir = "C:\\Program Files\\containerd\\cni\\conf"
+```
+
+```powershell
+# containerd 注册服务
+.\containerd.exe --register-service
+# 启动containerd
+Start-Service containerd
```
Comment (tengqm) [inline content/zh/docs/setup/production-environment/container-runtimes.md:876]: 1. 请注意空格
1. 请在译文中使用中文全角字符标点
1. 中文文字和英文文字之间留一个空格
1. 比较长的行需要断行
1. 请检查文字（如 867)
1. 注意英文文字的大小写，如 windows -> Windows，cni -> CNI

Comment (Yuaninga) [reply content/zh/docs/setup/production-environment/container-runtimes.md:872]: 好了

**Pull request merged**